nums/core/compute/numpy_compute.py

```diff
@@ -282,6 +282,25 @@ def advanced_assign_block_along_axis(
             dst_arr[tuple(dst_sel)] = src_arr[tuple(src_sel)]
         return dst_arr
 
+    def triu_copy(self, arr, rows, cols):
```

Comment:
Can you explain what's going on here? is `arr.shape != (rows, cols)`? If it's equal, we can simply use `arr.copy()`, and in that case we ought to add a kernel op for copy.